etl/app/server/database.js

```diff
@@ -893,13 +561,43 @@ export async function trimNationalDownloads(pool) {
   });
 }
 
+// Build the query for creating the indexes
+function buildIndexQuery(overrideWorkMemory, tableName) {
+  const indexTableName = tableName.replaceAll('_', '');
+
+  let query = overrideWorkMemory
+    ? `SET maintenance_work_mem TO '${overrideWorkMemory}';\n`
+    : '';
+
+  const table = Object.values(tableConfig).find(
+    (table) => table.tableName === tableName,
+  );
+  if (!table) return '';
+
+  table.columns.forEach((column) => {
+    if (column.skipIndex) return;
+
+    const sortOrder = column.indexOrder || 'asc';
+    const collate = column.type ? '' : 'COLLATE pg_catalog."default"';
+    query += `
+      CREATE INDEX IF NOT EXISTS ${indexTableName}_${column.name}_${sortOrder}
+        ON ${tableName} USING btree
+        (${column.name} ${collate} ${sortOrder} NULLS LAST)
+        TABLESPACE pg_default;
+      COMMIT;
```

Comment:
This is causing a warning in the PostgreSQL logs. Since we're not executing each `profileEtl` as a transaction, and instead aborting the ETL upon any error, we can remove this.

Reply:
I changed this so that we fire off individual queries for creating the indexes. This just makes it so that we can check on the progress of indexes being created for a table. If we just remove the `COMMIT` line, then we can't see any indexes for a table until all of them have been created.